Frontend/ui-customer/src/Screens/RentalBookingFlow/RideScheduledScreen/Controller.purs

```diff
@@ -129,7 +129,7 @@ eval (GetBookingList resp) state =
       }
 
 eval CheckFlowStatusAction state = do
-  void $ pure $ toast$ getString STR.NO_RIDES_SCHEDULED_YET
+  -- void $ pure $ toast$ getString STR.NO_RIDES_SCHEDULED_YET
```

Comment:
Is this action required?

Reply:
Actually `CheckFlowStatusAction ` this is the default action for `getEstimateEvent`, so every time the polling ends this action is called and a toast was popping up, thus removing it.